backend/inspirehep/migrator/tasks.py

```diff
@@ -109,6 +110,37 @@ def migrate_record_from_legacy(recid):
     db.session.commit()
 
 
+def migrate_from_mirror_run_step(
+    disable_orcid_push=True, disable_references_processing=True, step_no=1
+):
+    """Allows to easily run step by step migration only for valid records """
+    if step_no == 0:
+        query = LegacyRecordsMirror.query.with_entities(LegacyRecordsMirror.recid)
+        query = query.filter(LegacyRecordsMirror.valid.is_(True))
```

Comment:
do we need to assignments for this?

Reply:
It's more readable than one long line.

Reply:
it's not IMO, the following looks better :)

``` python
LegacyRecordsMirror \
    .query.with_entities(LegacyRecordsMirror.recid) \
    .filter(LegacyRecordsMirror.valid.is_(True))
```

or 

```python
query = (
    LegacyRecordsMirror
    .query.with_entities(LegacyRecordsMirror.recid)
    .filter(LegacyRecordsMirror.valid.is_(True))
)
```

Reply:
Really? I don't see it but i can change it if it's that important for you ;)

Reply:
Black is forcing:
```
query = LegacyRecordsMirror.query.with_entities(
            LegacyRecordsMirror.recid
        ).filter(LegacyRecordsMirror.valid.is_(True))
```
And this looks terrible IMO

Reply:
yes please 👍 either option will work

Reply:
As i wrote Black is forcing 
```
query = LegacyRecordsMirror.query.with_entities(
            LegacyRecordsMirror.recid
        ).filter(LegacyRecordsMirror.valid.is_(True))
```
for both options. Is is really better?

Reply:
from assigning twice the same ``var`` it's better IMO :)

Reply:
If you say so. For me this what black is doing is less readable.

Reply:
we're using black exactly for that reason, not to have different ways to style things

Reply:
done